inference-engine/thirdparty/clDNN/src/program.cpp

```diff
@@ -937,7 +937,7 @@ bool program::extract_and_remove(program_node& node) {
         outputs.erase(std::remove(outputs.begin(), outputs.end(), &node), outputs.end());
 
         rename(node, "_cldnn_tmp_" + node_id);
-        rename(prev, node_id);
+        add_optimized_primitive_info(prev.id(), { node_id });
```

Comment:
Is it correct usage of function? As I can see, the first argument is `primitive_id optimized_primitive_id` and the second one is `std::vector<primitive_id> replaced_with_ids`, so you should swap the arguments